src/ontology/uberon-edit.obo

```diff
@@ -64,59 +64,59 @@ import: http://purl.obolibrary.org/obo/uberon/imports/pato_import.owl
 import: http://purl.obolibrary.org/obo/uberon/imports/pr_import.owl
 import: http://purl.obolibrary.org/obo/uberon/imports/ro_import.owl
 ontology: uberon/core
-property_value: dc-contributor "Alan Ruttenberg - https://orcid.org/0000-0002-1604-3078" xsd:string
-property_value: dc-contributor "Alexander Diehl - https://orcid.org/0000-0001-9990-8331" xsd:string
-property_value: dc-contributor "Anne Maglia - https://orcid.org/0000-0002-3467-2636" xsd:string
-property_value: dc-contributor "Anna Anagnostopoulos - https://orcid.org/0000-0002-6490-7723" xsd:string
-property_value: dc-contributor "Bgee Project - http://bgee.unil.ch" xsd:string
+property_value: dc-contributor https://orcid.org/0000-0002-1604-3078
+property_value: dc-contributor https://orcid.org/0000-0001-9990-8331
+property_value: dc-contributor https://orcid.org/0000-0002-3467-2636
+property_value: dc-contributor https://orcid.org/0000-0002-6490-7723
+property_value: dc-contributor https://www.wikidata.org/wiki/Q54985720
 property_value: dc-contributor "Bill Bug" xsd:string
 property_value: dc-contributor "Brian K Hall" xsd:string
-property_value: dc-contributor "Carlo Torniai - https://orcid.org/0000-0002-3734-1859" xsd:string
+property_value: dc-contributor https://orcid.org/0000-0002-3734-1859
 property_value: dc-contributor "Catherine Leroy" xsd:string
-property_value: dc-contributor "Ceri Van Slyke - https://orcid.org/0000-0002-2244-7917" xsd:string
-property_value: dc-contributor "Cynthia Smith" xsd:string
-property_value: dc-contributor "David Hill" xsd:string
-property_value: dc-contributor "David Osumi-Sutherland - https://orcid.org/0000-0002-7073-9172" xsd:string
+property_value: dc-contributor https://orcid.org/0000-0002-2244-7917
+property_value: dc-contributor https://orcid.org/0000-0003-3691-0324
```

Comment:
why not make `orcid` a prefix on the same level as the others?

Reply:
I think this has to do with typing, but I am not sure. We could try it. The way we represent the annotation in RDF is 

```
https://orcid.org/0000-0003-3691-0324^^xsd:anyUri
```

Not as an rdf resource. I am not certain of its reasons right of the bat. Its the same for seeAlso to github issues for example. @anitacaron, can you check in pato how it would look if you added a `prefix` declaration to pato-edit.owl, then open it in protege, then save it? Just what the diff would look like?

Reply:
See also: I also demonstrated using ORCID prefixes in https://github.com/oborel/obo-relations/pull/540 (still awaiting external validation)

Reply:
Seems not to work.. Sorry man!

Reply:
Schade. At least I tried!